app/src/main/java/de/danoeh/antennapod/dialog/PlaybackControlsDialog.java

```diff
@@ -99,7 +100,7 @@ private void setupUi() {
         final TextView txtvPlaybackSpeed = dialog.findViewById(R.id.txtvPlaybackSpeed);
         float currentSpeed = getCurrentSpeed();
 
-        txtvPlaybackSpeed.setText(String.format("%.2fx", currentSpeed));
```

Comment:
They were using `Locale.getDefault()` by default but Android lint, correctly, asks to make requested locale explicit.